wasm/Makefile.toml

```diff
@@ -0,0 +1,5 @@
+[tasks.wasm-build]
+description = "Build WASM target for deno_task_shell"
+command = "wasm-pack"
+install_crate = { crate_name = "wasm-pack", binary = "wasm-pack", test_arg = "-V" }
+args = ["build", "--target", "deno", "--scope", "@deno/deno_task_shell"]
```

Comment:
One open question is what the name of this WASM question should be. I think the name `@deno/deno_task_shell` is okay, but in reality we're only exposing the task parsing API so maybe it should be something that better conveys the scope like `@deno/shell-cmd-processing`